### PR TITLE
refactor(frontend): adjust for local API

### PR DESCRIPTION
**Pull request by b5**

### CODE REVIEW

File: frontend/src/features/deploy/state/deployActions.ts [hunk 1]
```diff
@@ -3,17 +3,41 @@ import { NewWorkflow, Workflow, workflowInfoFromWorkflow, workflowScriptString }
...
+// }
+
 export function deployWorkflow(w: Workflow): ApiActionThunk {
   return async (dispatch, getState) => {
     return dispatch({
       type: 'deploy',
       
       [CALL_API]: {
-        endpoint: 'deploy',
+        endpoint: '/auto/deploy',
```
Comment (b5) [inline frontend/src/features/deploy/state/deployActions.ts]: possible different cloud endpoint

File: frontend/src/features/dsPreview/state/dsPreviewActions.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ function fetchDsPreview (ref: QriRef): ApiAction {
     type: 'datasetpreview',
     ref,
     [CALL_API]: {
-      endpoint: 'dataset_summary',
+      endpoint: 'ds/get',
```
Comment (b5) [inline frontend/src/features/dsPreview/state/dsPreviewActions.ts]: cloud & local differ here. In my code spike I've adjusted `qri get` with no selector to return a preview

File: frontend/src/features/deploy/state/deployActions.ts (hunk 1 above)
Comment (Arqu) [reply frontend/src/features/deploy/state/deployActions.ts]: Cloud doesn't have this one yet. But there is probably a long list of endpoints that generally do need to be brought over from core into cloud too. I'll make an issue for myself to review the entire API spec and filter out the ones that need to be added.

File: frontend/src/features/dsPreview/state/dsPreviewActions.ts (hunk 2 above)
Comment (Arqu) [reply frontend/src/features/dsPreview/state/dsPreviewActions.ts]: q: Should this be the case? Shouldn't get return the "full" response while previews should be separate. We do also have `/remote/previews` albeit a bit different thing there. I guess using `/ds/get` to fetch previews should be fine once cursors and proper paging are set up. 

Still not sure if we should bend `get` into this. And am more for setting up `/ds/preview` for it for now. The route fix as requested can land as soon as the changes in core for `get` are on master.

**Comment by b5** [comment]
superseded by #206

**Pull request closed without merging**